dotcom-rendering/src/web/layouts/InteractiveLayout.tsx

```diff
@@ -93,6 +93,7 @@ const InteractiveGrid = ({ children }: { children: React.ReactNode }) => (
 						'.      border  lines'
 						'.      border  meta'
 						'body   body    body'
+						'bottomlines  bottomlines   bottomlines'
```

Comment:
we might want to remove this? (i think it was just as a test?)

Reply:
Ah yes!